.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,3 +8,29 @@ assignees: ''
 ---
 
 Please only report technical bugs. Clarify design issues with one of our designers: Ken Brossi, Jonas Güther and Thomas Veit
```

Comment:
I would suggest to remove this line. Because we are solving even design issues and because of the full person names here.

Reply:
I adjusted it. We just should be more strict.